tools/65-libwacom.rules.in

```diff
@@ -7,6 +7,9 @@ KERNEL!="event[0-9]*", GOTO="libwacom_end"
 ATTRS{name}=="* Consumer Control", GOTO="libwacom_end"
 ATTRS{name}=="* System Control", GOTO="libwacom_end"
 
+# HUION keyboard is not a tablet
+ATTRS{name}=="HUION * Keyboard", GOTO="libwacom_end"
+
```

Comment:
```suggestion
```
Let's see if this works, commit suggestion to remove this hunk so we can squash it together and merge